internal/cmd/postgresflex/instance/create/create.go

```diff
@@ -0,0 +1,266 @@
+package create
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	cliErr "github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/projectname"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/client"
+	postgreslexUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/spinner"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/postgresflex"
+	"github.com/stackitcloud/stackit-sdk-go/services/postgresflex/wait"
+)
+
+const (
+	instanceNameFlag   = "name"
+	aclFlag            = "acl"
+	backupScheduleFlag = "backup-schedule"
+	flavorIdFlag       = "flavor-id"
+	cpuFlag            = "cpu"
+	ramFlag            = "ram"
+	replicasFlag       = "replicas"
+	storageClassFlag   = "storage-class"
+	storageSizeFlag    = "storage-size"
+	versionFlag        = "version"
+	typeFlag           = "type"
+
+	defaultBackupSchedule = "0 0/6 * * *"
```

Comment:
was this checked with the team?

Reply:
you can ask @joaopalet why he decided to go for the hardcoded default version instead of fetching the latest one from the API. 

Fetching the latest would be the best I agree, but we should be careful to not introduce too many calls that would make the CLI slow

Reply:
At the time I used the default value confirmed with the team and didn't think about always getting the most recent value from the API

Reply:
Aligned defaults with Portal, for version it pulls the latest version from the API

Reply:
Will change for mongodbflex in a separate PR